src/encoded/schemas/file.json

```diff
@@ -223,22 +223,97 @@
         }
     },
     "facets": {
+        "access_status": {
+            "title": "Access"
+        },
+        "donors.display_title": {
+            "title": "Donor",
+            "search_type": "sayt_without_terms",
+            "sayt_item_type": "Donor"
+        },
+        "donors.age": {
+            "title": "Donor Age",
+            "aggregation_type": "range",
+            "number_step": 1,
+            "ranges": [
+                {
+                    "from": 0,
+                    "to": 9,
+                    "label": "0-9"
+                },
+                {
+                    "from": 10,
+                    "to": 19,
+                    "label": "10-19"
+                },
+                {
+                    "from": 20,
+                    "to": 29,
+                    "label": "20-29"
+                },
+                {
+                    "from": 30,
+                    "to": 39,
+                    "label": "30-39"
+                },
+                {
+                    "from": 40,
+                    "to": 49,
+                    "label": "40-49"
+                },
+                {
+                    "from": 50,
+                    "to": 59,
+                    "label": "50-59"
+                },
+                {
+                    "from": 60,
+                    "to": 69,
+                    "label": "60-69"
+                },
+                {
+                    "from": 70,
+                    "to": 79,
+                    "label": "70-79"
+                },
+                {
+                    "from": 80,
+                    "to": 89,
+                    "label": "80-89"
+                },
+                {
```

Comment:
Small note: I think all ages 89 and above will be set to 89, so we may want to have the range be "80-88" and "89 and up" or something similar

Reply:
I'll check with Elizabeth; I believe they wanted it to be 90+, but that might have changed.